bower.json

```diff
@@ -18,12 +18,16 @@
     "templates"
   ],
   "dependencies": {
-    "angular": "1.2.18",
+    "angular": "~1.3.0",
     "bootstrap-select": "1.5.4",
-    "common-style": "https://github.com/Rise-Vision/common-style.git",
-    "jquery": "1.11.0",
+    "jquery": "~2.1.4",
+    "rv-common-style": "https://github.com/Rise-Vision/common-style.git#1.3.14",
```

Comment:
@donnapep I don't think so. I'm for leaving the tag off and just take the latest release and let the Widgets specify particular tags.